workday/12.1/modules/ROOT/pages/workday-connector-studio.adoc

```diff
@@ -37,34 +37,38 @@ Add Workday Connector to your Mule project to automatically populate the XML cod
 Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
 
 [[configure-input-source]]
-== Configure an Input Source
+== Configure a Source
 
-An input source initiates a flow when a specified condition is met.
+A source (listener) initiates a flow when a specified condition is met.
```

Comment:
I know we talked about source listener, but I'm still not convinced since I usually make the connection of listener with HTTP listener...but devs could confirm if this is good for them.

Reply:
Gaston Brito is the one who gave me that terminology.